signup-front/src/components/organisms/form-sections/DemarcheSection/DemarcheSectionReadOnly.js

```diff
@@ -52,6 +52,37 @@ export const DemarcheSectionReadOnly = ({ scrollableId }) => {
                   modifiées.
                 </p>
               )}
+              {!isEmpty(modifiedScopes) && (
+                <div>
+                  <p>
+                    Les scopes suivants ont étés modifiés depuis le modèle
+                    pré-rempli:
```

Comment:
plz insert non breaking space before `:`

Reply:
replace *« scopes »* with one of the following:

- périmètre de données
- données

Reply:
*ont été~s~ modifiés*

Reply:
replace *« depuis le »* with *« par rapport au »* ?